### ISSUE TITLE
Faraday::Error::ResourceNotFound Did you mean? Faraday::ResourceNotFound

### ISSUE THREAD

**Issue by Souravgoswami** [dblock/iex-ruby-client#79]
Hi, if I search for some term that doesn't exist, I get:
```
Faraday::Error::ResourceNotFound Did you mean? Faraday::ResourceNotFound
```

So Instead of rescuing `Faraday::ResourceNotFound`, I have to rescue `NameError` now.

**Comment by dblock** [comment]
This is a Faraday versioning problem, either lock Faraday < 1, or upgrade the client to Faraday 1.0 similarly to https://github.com/dblock/open-weather-ruby-client/pull/13. Want to give it a shot in a PR?

**Comment by Souravgoswami** [comment]
I just wanted to send PR, but it's fixed before that... Thanks for the quick action!

**Comment by dblock** [comment]
It's not fixed AFAIK, we didn't do anything.

**Comment by dblock** [comment]
Oh I see, it was done part of https://github.com/dblock/iex-ruby-client/pull/71, we just need to do a release.

**Comment by dblock** [comment]
I released 1.2.0.
